source/integrate-with-integration-environment/prove-users-identity.html.md.erb

```diff
@@ -157,74 +166,92 @@ The JWT contains the following claims:
-| `expiryDate`     | The expiration date as an [ISO 8601 date][external.ISO-8601-date] string. |
+| `expiryDate`     | The expiration date as an [ISO 8601 date][external.ISO-8601-date] string.                                                                                                                                                                                                                               |
 
 
 ## Understand your user's driving licence claim
-The <code>https://vocab.account.gov.uk/v1/drivingPermit</code> claim contains the details of your user’s driving licence, if they submitted one when proving their identity.
+The <code>https://vocab.account.gov.uk/v1/drivingPermit</code> claim contains the details of your user’s driving
+licence, if they submitted one when proving their identity.
 
-| Property         | Definition         |
-|------------------|--------------------|
-| `expiryDate`     | The expiry date of the driving licence as an [ISO 8601 date][external.ISO-8601-date] string. |
+| Property         | Definition                                                                                                                                                                                                                            |
+|------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `expiryDate`     | The expiry date of the driving licence as an [ISO 8601 date][external.ISO-8601-date] string.                                                                                                                                          |
 | `issueNumber`    | The last 2 characters of the driving licence number – these show how many times the user has received a new driving licence. You'll only receive this property for licences issued by the Driver and Vehicle Licensing Agency (DVLA). |
-| `issuedBy`       | The organisation that issued the driving licence. |
-| `personalNumber` | The driver number of the driving licence. This is a string unique to the user. |
+| `issuedBy`       | The organisation that issued the driving licence.                                                                                                                                                                                     |
+| `personalNumber` | The driver number of the driving licence. This is a string unique to the user.                                                                                                                                                        |
+
+## Understand your user's National Insurance Number claim
+The <code>https://vocab.account.gov.uk/v1/socialSecurityRecord</code> claim contains the details of your user’s National
+Insurance Number, if they submitted it while proving their identity.
```

Comment:
In our style guide we have this as National Insurance number so can that be reflected here and above please